microprofile/microprofile-fault-tolerance-2.0/src/main/resources/project-defaults.yml

```diff
@@ -0,0 +1,19 @@
+swarm:
+  hystrix:
+    threadpool:
+      default:
+        # for asynchronous bulkhead tests
+        maximumSize: 40
+        allowMaximumSizeToDivergeFromCoreSize: true
+    command:
+      default:
+        fallback:
+          isolation:
+            semaphore:
+              # for synchronous bulkhead tests
+              maxConcurrentRequests: 20
+        execution:
+          isolation:
+            thread:
+              # we wait 5 seconds in the test, to trigger a failure behavior, but Hystrix allows just 1 sec by default
```

Comment:
If we wait 5 seconds in the test, should the timeout here be like 6 seconds or so? To account for time measurement inaccuracy.